packages/algo-ts/src/impl/state.ts

```diff
@@ -1,7 +1,8 @@
-import { bytes } from '../primitives'
+import { bytes, Uint64 } from '../primitives'
 import { Account } from '../reference'
+import { uint8ArrayToHex } from './encoding-util'
 import { AssertError } from './errors'
-import { BytesCls } from './primitives'
+import { BytesCls, Uint64Cls } from './primitives'
 
 export class GlobalStateCls<ValueType> {
```

Comment:
Should this class use the `app_global_get` and `app_global_put` etc ops underneath? No point storing the value internally here if those ops need access to the same data

Reply:
The implementation is now moved to the testing package, 759d18c.